package.json

```diff
@@ -16,8 +16,13 @@
     "babel-preset-react": "^6.5.0",
     "html-webpack-plugin": "^2.21.0",
     "react": "^15.1.0",
+    "react-bootstrap": "^0.29.5",
     "react-dom": "^15.1.0",
     "webpack": "^1.13.1",
     "webpack-dev-server": "^1.14.1"
+  },
+  "devDependencies": {
```

Comment:
If you'd like, we can spend some time tomorrow going over webpack loader stuff!!! What do you think? Tomorrow or some other time?